### PR TITLE
Fix wrong configuration name

### PR DESCRIPTION
**Pull request by 315157973**
### Motivation

fileSystemProfilePath : For File System Storage, file system profile path

offloadersDirectory: The directory for all the offloader implementations

### CODE REVIEW

File: site2/docs/tiered-storage-filesystem.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ You can configure filesystem offloader driver in the configuration file `broker.
     |---|---|---
     `managedLedgerOffloadDriver` | Offloader driver name, which is case-insensitive. | filesystem
     `fileSystemURI` | Connection address | hdfs://127.0.0.1:9000
-    `offloadersDirectory` | Hadoop profile path | ../conf/filesystem_offload_core_site.xml
+    `fileSystemProfilePath` | Hadoop profile path | ../conf/filesystem_offload_core_site.xml
```
Comment (Anonymitaet) [inline site2/docs/tiered-storage-filesystem.md:88]: @zymap could you please help review? Thanks

**Review by zymap — APPROVED** [review]
LGTM

**Comment by Anonymitaet** [comment]
@315157973 thanks for your contribution. Does this affect only master or other versioned docs? 
If latter, could you please help update all affected versions? Thanks

**Comment by 315157973** [comment]
> @315157973 thanks for your contribution. Does this affect only master or other versioned docs?
> If latter, could you please help update all affected versions? Thanks

All versions are fixed

**Pull request merged**